invenio_rdm_records/tokens/scopes.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""OAuth2 tokens scopes."""
+
+from invenio_i18n import lazy_gettext as _
+from invenio_oauth2server.models import Scope
+
+tokens_generate_scope = Scope(
```

Comment:
Is this necessary though? I saw that example, but since the scope is not visible in the UI, this is just a string that has to match.

Reply:
There are a couple of places besides the UI (e.g. `invenio tokens create ...`) that validate the scopes that are passed. The CLI specifically is actually the way we used to create these "hidden" tokens. I'm not sure also if the validation would silently fail (which would be bad in case of a typo), but in any case, it's better for consistency and testing as well (I missed actually in the tests that you're passing `_scopes=...` to the model, which bypasses the validation 😏 )

Reply:
Very good catch, thanks! Fixed both scopes and tests 🐀